cmd/cmds_test.go

```diff
@@ -224,6 +224,24 @@ func noConflictsWereReported(ctx context.Context) (context.Context, error) {
 	return ctx, nil
 }
 
+func conflictsAreReported(ctx context.Context) (context.Context, error) {
+	recipeStdout := ctx.Value(recipeStdoutCtxKey{}).(string)
+	recipeStderr := ctx.Value(recipeStderrCtxKey{}).(string)
```

Comment:
Should we have a step which checks that stderr is empty?

Reply:
Added in a8aebdce085ab68086b6e30182879899a888cf3f